src/reporter/template.hbs

```diff
@@ -544,19 +544,19 @@
             <h3>Comparison</h3>
             <div class="baseline-container">
               <img
-                src="../cypress-visual-screenshots/baseline/$__id__.png"
+                src="../cypress-image-diff-screenshots/baseline/$__id__.png"
                 alt="$__id__"
               />
             </div>
             <div class="diff-container">
               <img
-                src="../cypress-visual-screenshots/diff/$__id__.png"
+                src="../cypress-image-diff-screenshots/diff/$__id__.png"
                 alt="$__id__"
               />
             </div>
             <div class="comparison-container">
               <img
-                src="../cypress-visual-screenshots/comparison/$__id__.png"
+                src="../cypress-image-diff-screenshots/comparison/$__id__.png"
```

Comment:
Does the template work with custom SCREENSHOTS_DIR variable?

If I understand correctly we can set the screenshot folder with SCREENSHOTS_DIR variable, but not sure if the template processes SCREENSHOTS_DIR

Reply:
I don't think we should continue to add new features to the old report. That means the `SCREENSHOTS_DIR` and `REPORT_DIR` do not have any effect on the old report, which continues to function with the hardcoded `cypress-image-diff-screenshots` and `cypress-image-diff-html-report` directory. I'll note this in our documentation.